docs/http-template.md

```diff
@@ -19,16 +16,31 @@ spec:
   templates:
     - name: main
       steps:
-        - - name: good
+        - - name: get-google-homepage
             template: http
             arguments:
-              parameters: [{name: url, value: "https://raw.githubusercontent.com/argoproj/argo-workflows/4e450e250168e6b4d51a126b784e90b11a0162bc/pkg/apis/workflow/v1alpha1/generated.swagger.json"}]
+              parameters: [{name: url, value: "https://www.google.com"}]
     - name: http
       inputs:
         parameters:
           - name: url
       http:
-       # url: http://dummy.restapiexample.com/api/v1/employees
-       url: "{{inputs.parameters.url}}"
-      
+        timeoutSeconds: 20 # Default 30
+        url: "{{inputs.parameters.url}}"
+        method: "GET" # Default GET
+        headers:
+          - name: "x-header-name"
+            value: "test-value"
+        # Template will suceed if evaluated to true, otherwise will fail
+        # Available variables:
+        #  statusCode: int, the response status code
+        #  body: string, the response body
```

Comment:
I think I'd want to to `json(body).foo.bar`, is that possible?

Reply:
Will make sure it is